test/InverseSolve/testMisfitParams.jl

```diff
@@ -117,4 +125,11 @@ x1, = projGNCG(x0,pInv,pMis)
 
 pInv.maxIter = 5
 x2, = projGNCG(x0,pInv,pMisRefs)
-@test norm(x1-x2)/norm(x1) < 1e-12
\ No newline at end of file
+@test norm(x1-x2)/norm(x1) < 1e-12
+
+clear!(pMisRefs);
+for k=1:length(pMis)
+	clear!(pMis[k],clearPFor=true, clearData=true,clearMesh2Mesh=true);
```

Comment:
In principle, yes, the pMis array may be uninitialized. But we never do
that throughout the code...
In any case, this is just our test.. the change doesn't need to be here.
Also RemoteRef can point to a null information, so clear! in Utils is also
not "protected" in this sense.
I don't know how NULL is defined in julia.

On Tue, Apr 19, 2016 at 1:03 PM, Lars Ruthotto <notifications@github.com>
wrote:

> In test/InverseSolve/testMisfitParams.jl
> <https://github.com/JuliaInv/jInv.jl/pull/4#discussion_r60300839>:
>
> > \ No newline at end of file
> > +@test norm(x1-x2)/norm(x1) < 1e-12
> > +
> > +clear!(pMisRefs);
> > +for k=1:length(pMis)
> > +	clear!(pMis[k],clearPFor=true, clearData=true,clearMesh2Mesh=true);
>
> should we check if the fields in pMis are empty?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/JuliaInv/jInv.jl/pull/4/files/1937679c913da380fa95198e360422e91128f3ea#r60300839>
>